src/interface_r/R/data_prep.R

```diff
@@ -3,10 +3,11 @@
 #'@param data_table `data.table` object containing data that needs to be preprocessed for h2o4gpu
 #'@param response Response column as a string or index
 #'@param save_csv_path Path to save processed data as a csv
+#'@param max_label_encoding_levels The maximum number of uniques required in a column to consider it a categorical variable. Default is 1000
 #'@export
```

Comment:
You can remove "Default is 1000" since users can see the default value in help pages